src/controllers/cohort.js

```diff
@@ -21,3 +25,17 @@ export const getAll = async (req, res) => {
     sendMessageResponse(res, 500, 'Unable to fetch Cohorts')
   }
 }
+
+export const getById = async (req, res) => {
+  const Id = parseInt(req.params.id)
+  const foundCohort = await getCohortById(Id)
+  return sendDataResponse(res, 200, { foundCohort })
```

Comment:
could you change this to `return sendDataResponse(res, 200, { cohort: foundCohort })` please